### ISSUE TITLE
[Usage]: UniMoE-2.5（基于Qwen3-VL-MoE的派生SFT模型）适配vllm-ascend:v0.19.1rc1

### ISSUE THREAD

**Issue by zyxzyxzyx** [vllm-project/vllm-ascend#9451]
### Your current environment

昇腾910B2
推理采用官方镜像环境quay.io/ascend/vllm-ascend:v0.19.1rc1

### How would you like to use vllm on ascend

**现象:**
移植代码侧排查未发现明显bug，vllm版和hf版在v0.19的同容器内对部分prompt的输出不一致

<img width="933" height="805" alt="Image" src="https://github.com/user-attachments/assets/b26e7905-1d13-48b3-bb1a-e51d32d482bd" />

具体输出结果见附件：hf_result.log、vllm_result.log

**模型信息：**
见附件：*.json文件
ckpt单safetensors shard，共1325个权重，总参数量~9B，bf16

**目前问题：**
希望能帮忙看看，这样的结果是正常的吗？还是说适配方案上我这边还有问题，如果是适配问题麻烦给出一些方向性建议，谢谢

**附件：**
[UniMoE-2.5模型适配vllm-ascend_v0.19.1rc1_report.zip](https://github.com/user-attachments/files/28137541/UniMoE-2.5.vllm-ascend_v0.19.1rc1_report.zip)

**Comment by linfeng-yuan** [comment]
Could you please attach your adaptation code with a PR? Additionally, it is recommended to add debug code in `model_runner.py` to print `input_ids` and output logits and compare them with those from HF.
